src/test/javascript/spec/service/orion/orion-version-validator.service.spec.ts

```diff
@@ -51,52 +44,65 @@ describe('OrionValidatorService', () => {
         });
         orionVersionValidator = TestBed.inject(OrionVersionValidator);
 
-        profileInfoStub = stub(TestBed.inject(ProfileService), 'getProfileInfo');
-        profileInfoStub.returns(of(profileInfo));
+        profileInfoStub = jest.spyOn(TestBed.inject(ProfileService), 'getProfileInfo');
+        profileInfoStub.mockReturnValue(of(profileInfo));
 
-        navigateSpy = spy(TestBed.inject(Router), 'navigateByUrl');
+        navigateSpy = jest.spyOn(TestBed.inject(Router), 'navigateByUrl');
     });
 
     afterEach(() => {
-        sinon.restore();
+        jest.restoreAllMocks();
     });
 
     it('should route to the error page if a legacy version is used', () => {
         setUserAgent(userAgent + legacy);
         orionVersionValidator.isOrion = true;
 
-        orionVersionValidator.validateOrionVersion();
+        orionVersionValidator.validateOrionVersion().subscribe((result) => expect(result).toBe(false));
```

Comment:
I think we already had this at some similar point but I think this might be different. Could you verify that this is actually called in time?

Reply:
If a fix is necessary, please adapt this for other occurrences.

Reply:
It actually isnt. Thanks, totally missed that. I added the missing tick()s